packages/transformer-directives/package.json

```diff
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "@unocss/core": "workspace:*",
+    "@unocss/preset-mini": "workspace:*",
```

Comment:
We shouldn't rely on the preset. I think we could move them to a new package `@unocss/utils`